Docs/XCHammerFAQ.md

```diff
@@ -0,0 +1,39 @@
+## XCHammer Frequently Asked Questions
+
+## Why XCHammer?
+
+Many features in Xcode rely on the fact that Xcode is both the editor and build
+system. Since XCHammer builds Bazel projects both Xcode and Bazel as a core
+design goal, it supports all of these features by default.
+
+_XCHammer generates an Xcode project from Bazel rules, which we've found to be a
+great way to describe an Xcode project and corresponding apps!_
+
+## How is XCHammer versioned?
+
+XCHammer releases correspond to a tested, canonical, Bazel release.
```

Comment:
XCHammer aims to be compatible with the latest official version of Bazel, rules_apple and rules_swift.